uplink/config.go

```diff
@@ -150,3 +156,47 @@ func (c Config) GetEncryptionScheme() storj.EncryptionScheme {
 		BlockSize: int32(c.Enc.BlockSize),
 	}
 }
+
+// LoadEncryptionKey loads the encryption key stored in the file pointed by
+// filepath.
+//
+// An error is file is not found or there is an I/O error.
+func LoadEncryptionKey(filepath string) (key *storj.Key, error error) {
+	if filepath == "" {
+		return &storj.Key{}, nil
+	}
+
+	file, err := os.Open(filepath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("not found key file %q", filepath)
+		}
+
+		return nil, err
+	}
+
+	defer func() { err = errs.Combine(err, file.Close()) }()
+
+	// TODO: if/v3-1541#3 should we check if the file contains a values of length
+	// storj.KeySize??
+	key = &storj.Key{}
+	if _, err := file.Read(key[:]); err != nil && err != io.EOF {
+		return nil, err
+	}
```

Comment:
Same here, shouldn't we use `storj.NewKey(<content-of-file>)`?

Reply:
Because of the same than commented above, the file contains the key to use, not a passphrase

Reply:
This comment is still valid as per our last discussion.

Reply:
What the right size to use here to read the content file?
Should we read the file up to the end?

Reply:
For now, I've used 1024 bytes, however depending of your answers, I will have to change it.

Reply:
Just use https://golang.org/pkg/io/ioutil/#ReadFile